packages/labs/src/components/omnibox/omnibox.tsx

```diff
@@ -179,20 +184,32 @@ export class Omnibox<T> extends React.Component<IOmniboxProps<T>, IOmniboxState<
     }
 
     private maybeRenderMenu(listProps: IQueryListRendererProps<T>) {
-        if (this.state.query.length > 0) {
+        const { initialContent } = this.props;
+        let menuChildren: any;
+
+        if (!this.isQueryEmpty()) {
+            menuChildren = this.renderItems(listProps);
+        } else if (initialContent != null) {
+            menuChildren = initialContent;
+        }
+
+        if (menuChildren != null) {
```

Comment:
nit: i tend to do:

```tsx
return menuChildren == null
    ? undefined
    : (
        <Menu ulRef={listProps.itemsParentRef}>
            {this.renderItems(listProps)}
            {menuChildren}
        </Menu>
    );
```

or maybe even:

```tsx
return menuChildren == null ? undefined : (
    <Menu ulRef={listProps.itemsParentRef}>
        {this.renderItems(listProps)}
        {menuChildren}
    </Menu>
);
```

Reply:
Happy to address those nits but they seem to be conflicting with some previous feedback. @giladgray?

Reply:
let's just switch to prettier and throw away all these debates